app/models/event.rb

```diff
@@ -1,9 +1,18 @@
 # frozen_string_literal: true
 
+# Event model class to manage all the
+# events data (validations and relationships).
```

Comment:
Hey I know youre learning but in my opinion you shouldnt add comments like this 
They will eventually bloat your code
You can checkout this [video](https://www.youtube.com/watch?v=Bf7vDBBOBUA)

Reply:
Got it! Thanks for the advice!